uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -125,7 +125,8 @@ define([
             request_type: 'leave',
             uploader: FileUpload.uploader({
               entityTable: 'civicrm_hrleaveandabsences_leave_request',
-              crmAttachmentToken: sharedSettings.attachmentToken
+              crmAttachmentToken: sharedSettings.attachmentToken,
+              queueLimit: 5 //allow to upload upto 5 files for now
```

Comment:
Can this property be encapsulated in the `FileUpload` service? it seems something more of a generic config setting rather than something specific that `LeaveRequestInstance` should be concerned about